nodejs/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:18.12.1
```

You could use `node:18.12.1-alpine`, which is a lot more lightweight (based on Alpine Linux instead of Debian/Ubuntu), and does the job just as well.

```suggestion
FROM node:18.12.1-alpine
```